### PR TITLE
add flag in cluster to create dir

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
add -p to create directory
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/gce/gci/mounter/stage-upload.sh [hunk 1]
```diff
@@ -51,8 +51,8 @@ function cleanup {
 # Delete temporary directories on exit
 trap cleanup EXIT
 
-mkdir ${RKT_DIR}
-mkdir ${ACI_DIR}
+mkdir -p ${RKT_DIR}
+mkdir -p ${ACI_DIR}
```
Comment (mikedanese) [inline cluster/gce/gci/mounter/stage-upload.sh]: This isn't necessary. Why should we add it?

Comment (jianglingxia) [reply cluster/gce/gci/mounter/stage-upload.sh]: sorry,I have update it!

**Comment by jianglingxia** [comment]
thanks @bowei but maybe need @mikedanese /approve no-issue the robot can label approve label

**Comment by mikedanese** [comment]
this code was removed

**Pull request closed without merging**